src/components/Detail/Detail.scss

```diff
@@ -0,0 +1,135 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:400,700');
+$font: 'Lato', sans-serif;
+
+$white: #F5F5F5;
```

Comment:
You can put colors in a separate, "global" .scss file, for example named `constants.scss` which will contain all universal values such as colors, so you can import that in all other .scss files with:
`@import 'global/constants'`